src/parcsr_ls/par_ilu.h

```diff
@@ -7,17 +7,51 @@
 
 #ifndef hypre_ParILU_DATA_HEADER
 #define hypre_ParILU_DATA_HEADER
+
+#include "_hypre_utilities.hpp"
```

Comment:
@liruipeng Is this the right place to put the `.hpp` header?

Reply:
@rfalgout @oseikuffuor1 I would avoid including `.hpp` in in a `.h` but instead include in `.c` that needs CUDA